### PR TITLE
Add SpoilerOpened tracking event, for saving in PlanetScale

### PR DESCRIPTION
**Pull request by shn-srl**
Sends a SpoilerOpened tracking event to Planetscale

To confirm:

[ ] Is all the required information being saved ?

- Important: PlanetScale migration also needs to run

https://app.planetscale.com/serlo/serlo_planetscale/test-spoiler-tracking/

image

### CODE REVIEW

File: packages/editor/src/plugins/spoiler/static.tsx [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
   const { title, content } = state
+  const [hasSentSpoilerTrackingEvent, setHasSentSpoilerTrackingEvent] = useState(false)
+
+  const ab = useAB()
+
+  const trackSpoilerOpened = () => {
+    if(!ab || hasSentSpoilerTrackingEvent) return
+    // send tracking event
+    abSubmission({
+      entityId: -1, // should this be a real id?
```
Comment (shn-srl) [inline packages/editor/src/plugins/spoiler/static.tsx]: This needs to be double checked.

If entity id is needed, what id is it? I see string Ids but the abSubmission expects an int

**Review by Entkenntnis — CHANGES_REQUESTED** [review]
Two minor things:

- Can you revert the db change?
- Check if entity id is one of the three we use this event for

**Comment by shn-srl** [comment]
> Two minor things:
> 
>     * Can you revert the db change?
> 
>     * Check if entity id is one of the three we use this event for

DB change was on a development branch, which is now deleted :heavy_check_mark: 

Added EntityId check :heavy_check_mark:

**Review by Entkenntnis — APPROVED** [review]

**Review by Entkenntnis — CHANGES_REQUESTED** [review]
move code to integration and out of editor repo

**Review by Entkenntnis — APPROVED** [review]
Thank you

**Pull request merged**